optimism/src/keccak/tests.rs

```diff
@@ -499,28 +497,28 @@ fn test_keccak_prover() {
         let mut keccak_env = KeccakEnv::<Fp>::new(0, &preimage);
 
         // Keep track of the constraints and lookups of the sub-circuits
-        let mut keccak_circuit = KeccakCircuit::<Fp>::new(domain_size, &mut keccak_env);
+        let mut keccak_circuit = KeccakTrace::<Fp>::new(domain_size, &mut keccak_env);
 
-        // We want to use domain_size rows, even if that is an incomplete Keccak execution
         while keccak_env.constraints_env.step.is_some() {
             let step = keccak_env.constraints_env.step.unwrap();
+
             // Run the interpreter, which sets the witness columns
             keccak_env.step();
-            // Check witness satisfies constraints
-            keccak_env.witness_env.constraints();
```

Comment:
Why do you remove?

Reply:
Because I have other tests checking the witness data satisfies constraints, and it seemed unnecessary to do it again. Here the important part is checking the prover can generate a proof for the constraints instead.